monkey/monkey_island/cc/services/config.py

```diff
@@ -281,3 +321,66 @@ def decrypt_ssh_key_pair(pair, encrypt=False):
             pair['public_key'] = encryptor.dec(pair['public_key'])
             pair['private_key'] = encryptor.dec(pair['private_key'])
         return pair
+
+    @staticmethod
+    def add_PBA_files(post_breach_files):
```

Comment:
It's very unclear by documentation where the information is taken from and where it is saved.
Additionally, with the current name, you would expect `remove_PBA_files` would do the reverse operation with a similar function signature